aspnetcore/security/key-vault-configuration.md

```diff
@@ -210,25 +210,24 @@ config.AddAzureKeyVault(
 
 | Property         | Description |
 | ---------------- | ----------- |
-| `Client`         | <xref:Microsoft.Azure.KeyVault.KeyVaultClient> to use for retrieving values. |
-| `Manager`        | <xref:Microsoft.Extensions.Configuration.AzureKeyVault.IKeyVaultSecretManager> instance used to control secret loading. |
+| `Client`         | <xref:Azure.Security.KeyVault.Secrets.SecretClient> to use for retrieving values. |
+| `Manager`        | <xref:Azure.Extensions.Aspnetcore.Configuration.Secrets.KeyVaultSecretManager> instance used to control secret loading. |
 | `ReloadInterval` | `Timespan` to wait between attempts at polling the key vault for changes. The default value is `null` (configuration isn't reloaded). |
-| `Vault`          | Key vault URI. |
 
 ## Use a key name prefix
 
-<xref:Microsoft.Extensions.Configuration.AzureKeyVaultConfigurationExtensions.AddAzureKeyVault*> provides an overload that accepts an implementation of <xref:Microsoft.Extensions.Configuration.AzureKeyVault.IKeyVaultSecretManager>, which allows you to control how key vault secrets are converted into configuration keys. For example, you can implement the interface to load secret values based on a prefix value you provide at app startup. This allows you, for example, to load secrets based on the version of the app.
+AddAzureKeyVault provides an overload that accepts an implementation of <xref:Azure.Extensions.Aspnetcore.Configuration.Secrets.KeyVaultSecretManager>, which allows you to control how key vault secrets are converted into configuration keys. For example, you can implement the interface to load secret values based on a prefix value you provide at app startup. This allows you, for example, to load secrets based on the version of the app.
```

Comment:
Why were the xrefs removed?

Reply:
There is no new API reference Documentation about AddAzureKeyVault(), so in this PR only remove the old API.